### PR TITLE
Update to v2.3.2.post1

### PR DESCRIPTION
**Pull request by marcoesters**
Update to version 2.3.2.post1

Jira ticket [PKG-146](https://anaconda.atlassian.net/jira/software/c/projects/PKG/issues/PKG-146)

# Package Info

Home: https://facelessuser.github.io/soupsieve
Development: https://github.com/facelessuser/soupsieve
Documentation: https://facelessuser.github.io/soupsieve/
Git diff: https://github.com/facelessuser/soupsieve/compare/2.3.1...2.3.2.post1

### CODE REVIEW

**Review by varlackc — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,40 +1,38 @@
...
   run:
-    - python >=3.6
+    - python
 
 test:
   imports:
     - soupsieve
   requires:
-    - python <3.10
+    - python
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: We can remove `python` from `test/requires` because it's already in `run`

**Comment by skupr-anaconda** [comment]
This package is an indirect dependency of `conda-build` but I didn't see any breaking changes https://github.com/facelessuser/soupsieve/compare/2.3.1...2.3.2.post1 so we can update it

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,40 +1,38 @@
...
     - pip
     - beautifulsoup4
   commands:
     - pip check
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:32]: Is it make sense to add `run_test.py` with examples from the home site https://facelessuser.github.io/soupsieve/ to check if the package works correctly?

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**